### PR TITLE
4616: Integration test empty get

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#4616](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4616)
Task: [AB#7371](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7371)

### CODE REVIEW

File: tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Ordering Party Get.feature [hunk 1]
```diff
@@ -30,30 +31,36 @@ Scenario: 1. Get an ordering party
         | Fred      | Robinson | Fred.robinson@email.com | 12312543212     |
 
 @4616
-Scenario: 2. A non existent orderId returns not found
+Scenario: 2. Get an empty ordering party
+    When the user makes a request to retrieve the ordering-party section with the ID C000014-02
+    Then a response with status code 200 is returned
+    And an empty object is returned
```
Comment (sam-roberts-bjss) [inline tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Ordering Party Get.feature:37]: The line "And an empty object is returned" is more meaningful to a developer than a BA. What do you think to changing the phrase to a more business orientated phrase e.g. "And the response contains no data"

File: tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/Common/CommonSteps.cs [hunk 2]
```diff
@@ -39,6 +39,13 @@ public async Task ThenTheResponseContainsTheFollowingErrors(Table table)
             actual.Should().BeEquivalentTo(expected);
         }
 
+        [Then(@"an empty object is returned")]
+        public async Task ThenAnEmptyObjectIsReturned()
+        {
+            var response = (await _response.ReadBodyAsJsonAsync());
```
Comment (sam-roberts-bjss) [inline tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/Common/CommonSteps.cs:45]: Are the brackets needed around the code await `_response.ReadBodyAsJsonAsync()`?

Comment (RuthBerry96) [reply tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/Common/CommonSteps.cs]: 👍

File: tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Ordering Party Get.feature (hunk 1 above)
Comment (RuthBerry96) [reply tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Ordering Party Get.feature]: 👍

**Pull request merged**